bucket.go

```diff
@@ -370,10 +370,11 @@ func (b *Bucket) NextSequence() (uint64, error) {
 }
 
 // ForEach executes a function for each key/value pair in a bucket.
+// If the value is nil, then it's a nested bucket, and the key is the bucket name.
```

Comment:
This seems not correct. FYI. https://github.com/etcd-io/bbolt/issues/521#issuecomment-1578367675